Cargo.toml

```diff
@@ -32,6 +32,7 @@ members = [
   "bindings/java",
   "bindings/haskell",
   "bindings/lua",
+  "bindings/dotnet",
```

Comment:
What's the difference between `C#` and `dotnet`? `F#` can use `opendal-dotnet` directly?

Reply:
> F# can use opendal-dotnet directly?

Yes. I think this is the idiom in .NET world.